modules/edit-modes/src/lib/draw-rectangle-using-three-points-mode.ts

```diff
@@ -16,7 +17,7 @@ export class DrawRectangleUsingThreePointsMode extends ThreeClickPolygonMode {
       properties: {
         shape: 'Rectangle',
       },
-      geometry: {
+      geometry: rewind({
```

Comment:
Can you make this more generic by adding the rewind call in `ThreeClickPolygonMode` where it calls `getThreeClickPolygon`?

Reply:
I should be more explicit. Can you make this change in `ThreeClickPolygonMode.handleClick` instead? 

this:
```
const editAction = this.getAddFeatureOrBooleanPolygonAction(tentativeFeature.geometry, props);
```

can become
```
const editAction = this.getAddFeatureOrBooleanPolygonAction(rewind(tentativeFeature.geometry), props);
```

Then every mode that extends `ThreeClickPolygonMode` won't need to change.